lib/oli/communities/community.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Oli.Communities.Community do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Oli.Repo
+
+  schema "communities" do
+    field :name, :string
+    field :description, :string
+    field :key_contact, :string
```

Comment:
@jrissler should we verify if this is a user relationship instead of a string value?

Reply:
good as is for now - double checked.